.github/workflows/generateTranslations.yml

```diff
@@ -3,28 +3,155 @@ name: Generate static translations
 on:
   pull_request:
     types: [opened, synchronize]
-    paths-ignore: ['docs/articles/**/*.md', 'docs/redirects.csv', 'docs/assets/images/**']
+    paths: ['src/languages/en.ts']
```

Comment:
Not sure if we also want to run this when the translation script or prompts change, but I think this is probably fine for now. If those change, whoever's changing it can run it locally, targeting whatever paths they want to refine.